modules/rte_mis_school/rte_mis_school.libraries.yml

```diff
@@ -11,3 +11,8 @@ cshs_validation:
   dependencies:
     - core/jquery
     - core/once
+# For school_details pdf.
+school_details_pdf:
+  css:
+    theme:
+      css/school-details-style.css: {}
```

Comment:
We will have to add SASS and then convert it to CSS

Reply:
Updated